rust/processor/src/utils/counters.rs

```diff
@@ -147,6 +147,15 @@ pub static PROCESSED_BYTES_COUNT: Lazy<IntCounterVec> = Lazy::new(|| {
     .unwrap()
 });
 
+pub static BATCH_SIZE: Lazy<HistogramVec> = Lazy::new(|| {
```

Comment:
we already have a metric like this. it's `NUM_TRANSACTIONS_PROCESSED_COUNT`

Reply:
There is a slight semantic difference because this is `batch_end-batch_start` but for all practical purpose, this is good. Removed the counter for now.